src/processors/data_processor.py

```
@@ -0,0 +1,640 @@
+            # totalファイルが元データにある場合、計算結果と一致するか検証
+            if total_file and male_file and female_file and not self._is_empty_data_file(total_file):
+                self._verify_total_calculation(male_file, female_file, total_file, metadata)
+
+            logger.info(f"定点監視処理成功: {source_file.name} → {len(output_files)}ファイル")
+
+            # ログ記録
+            self._log_processing(source_file, output_files, metadata)
+
+            return NormalizationResult(success=True, source_path=source_file, output_files=output_files)
+
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)